docs/release-notes/self-hosted/self-hosted-v5.0.0.md

```diff
@@ -0,0 +1,93 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v5.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/3.9.16
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/4.0.28
+---
+
+# Self-hosted v5.0.0
+
+These release notes are for [Codacy Self-hosted v5.0.0](https://github.com/codacy/chart/releases/tag/5.0.0){: target="_blank"}, released on December 14, 2021. <!-- TODO Update release date -->
+
+To upgrade Codacy, follow [these instructions](../../chart/maintenance/upgrade.md).
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Breaking changes
+
+This version of Codacy Self-hosted no longer includes support for legacy manual organizations.
+
+## Product enhancements
+
+-   Now, Codacy supports linting OpenAPI and AsyncAPI descriptions in either YAML or JSON files using [Spectral](https://stoplight.io/open-source/spectral/){: target="_blank"}. (CY-5088)
+
+-   You can now [use an organization coding standard](https://docs.codacy.com/v5.0/organizations/using-a-coding-standard/) to apply the same coding best practices, conventions, or security rules to a group of repositories. (CY-4654)
+
+    ![Organization coding standard](../images/cy-4654.png)
+
+## Bug fixes
+
+-   Fixed an issue that prevented non-logged users to see the file list of open-source repositories. (CY-5298)
+-   Improved the visual feedback for the Jira integration status. (CY-5190)
+-   Fixed an issue that prevented the message "Refresh the page to see the results" from being displayed on the commit and pull request pages after a re-analysis was completed. (CY-5187)
+-   Fixed some default regular expressions on [<span class="skip-vale">codacy-checkstyle</span>](https://github.com/codacy/codacy-checkstyle){: target="_blank"} that could cause the code pattern PackageName to report false positives when configured using the Codacy UI. (CY-5185)
+-   Fixed an issue that caused ESLint to report false positive issues regarding import statement organization if prettier was enabled. (CY-5143)
+-   Fixed security vulnerabilities in the underlying Docker images for analysis workers. (CY-5140)
+-   Fixed an issue that could allow triggering requests from the Codacy instance servers by injecting a URL into the cursor parameter of the API endpoint to list organization repositories. (CY-5139)
+-   Added protection against CSRF attacks targeting the Codacy API v3. In the case of phishing, even if the victim opens a malicious link, the attack won't work. CVSS v3.1 score: 6.4 (Medium) (CY-5131)
+-   Fixed an issue that prevented Codacy from displaying secondary email addresses from GitLab accounts. (CY-5130)
+-   Fixed the issue cards in file lists that didn't display the issue category. (CY-5125)
```

Comment:
not in the files list, it's in the file page 👀

Reply:
@inesgomas I thought this fix was on a component that was used not only on the page for a file but also on any file list such as the files with issues on commits or pull requests. I'm going to fix it. :+1: